app/components/DocumentExplorer.tsx

```diff
@@ -258,42 +292,55 @@ function DocumentExplorer({ onSubmit, onSelect, items }: Props) {
   };
 
   const handleKeyDown = (ev: React.KeyboardEvent<HTMLDivElement>) => {
-    switch (ev.key) {
-      case "ArrowDown": {
-        ev.preventDefault();
-        setActiveNode(next());
-        break;
-      }
-      case "ArrowUp": {
-        ev.preventDefault();
-        if (activeNode === 0) {
-          focusSearchInput();
-        } else {
-          setActiveNode(prev());
-        }
-        break;
-      }
-      case "ArrowLeft": {
-        if (!searchTerm && isExpanded(activeNode)) {
-          toggleCollapse(activeNode);
-        }
-        break;
-      }
-      case "ArrowRight": {
-        if (!searchTerm) {
-          toggleCollapse(activeNode);
-        }
-        break;
-      }
-      case "Enter": {
-        if (isModKey(ev)) {
-          onSubmit();
-        } else {
-          toggleSelect(activeNode);
-        }
-        break;
-      }
+    if (ev.key === "ArrowDown" || ev.key === "ArrowUp") {
+      ev.preventDefault();
```

Comment:
Forced to put this out of `throttle` since then there seemed to be no way preventing it, no way I could think of at the very least.